### PR TITLE
recast S30 as product of reals, plus a couple edits to traits

### LINKED ISSUE #986: Space Suggestion: Use "Fréchet space R^ω" as the default name for S30
## Space Suggestion

Use "Fréchet space $\mathbb R^\omega$" as the default name for [S30](https://topology.pi-base.org/spaces/S000030).

## Rationale

While there are numerous aliases and characterizations for this space, the most elegant (in terms of the topological properties we model in $\pi$-Base) is probably the Tychonoff product of countably-many copies of the real line.

See also discussion at #956.

### PR DESCRIPTION
**Pull request by StevenClontz**
Closes #986

### CODE REVIEW

**Comment by GeoffreySangston** [comment]
Pseudocompact is derived now.

**Comment by StevenClontz** [comment]
Thanks - while I'm touching this I can remove those redundant traits

**Comment by StevenClontz** [comment]
@GeoffreySangston - all three suggestions were accurate and now implemented.

This is the result of some discussion at #986; can we get two approvals before merging?

**Review by GeoffreySangston — COMMENTED** [review]
I confirmed that the deleted traits are derived in this branch on pi-base.

I checked the reference to Willard for countable product of completely metrizable spaces.

I'm sorry I'm late to the party on Fréchet space. I see that Steen-Seebach has this as the name of a certain complete metric space. I haven't spent a lot of time studying metric spaces and don't know if Fréchet product metric is common or not, so I'll trust your judgement. Just submitting this as a comment since I didn't participate in the Fréchet space name discussion.

**Comment by prabau** [comment]
P55: item 7 in S&S should refer to space 37 instead of 36.

**Review by prabau — APPROVED** [review]
leaving it up to you if you want the minor tweak to P55

File: spaces/S000030/properties/P000055.md
Comment (StevenClontz) [inline spaces/S000030/properties/P000055.md:16]:
```suggestion
See item #7 for space #36/#37 in {{doi:10.1007/978-1-4612-6290-9_6}}.
```

It appears that #36 and #37 don't have separate listings in *Counterexamples* (because they recognized the homeomorphism) so perhaps it's clearer to use `#36/#37` in our references.

File: spaces/S000030/properties/P000026.md
Comment (StevenClontz) [inline spaces/S000030/properties/P000026.md:13]:
```suggestion
Or, see item #2 for space #36/#37 in {{doi:10.1007/978-1-4612-6290-9_6}}.
```

**Comment by StevenClontz** [comment]
Thanks @prabau. @david20000813 I'd like you to chime in before merging.

File: spaces/S000030/properties/P000055.md
Comment (prabau) [inline spaces/S000030/properties/P000055.md:16]:
```suggestion
See item #7 for space #37 in {{doi:10.1007/978-1-4612-6290-9_6}}.
```

**Review by prabau — CHANGES_REQUESTED** [review]

**Comment by prabau** [comment]
Space 37 is the product of reals, space 36 is the Hilbert space.  And item 7 is specifically about the product of reals, i.e., space 37.  So we should use that.

Note that we do the same for many other sections of S&S that describe multiple spaces.  If an item in the section refers to a particular space, we always refer in pi-base to that particular space number when referring to that item.

**Review by david20000813 — APPROVED** [review]

**Comment by david20000813** [comment]
Everything looks fine to me as well. Though, I agree with @prabau 's comment: completely metrizable is proved twice, once in item 1 for the Hilbert space, once in item 7 for the Fréchet space. It would be better to be clearer about this and just mention space 37 when referring to item 7.

**Comment by StevenClontz** [comment]
> Space 37 is the product of reals, space 36 is the Hilbert space. And item 7 is specifically about the product of reals, i.e., space 37. So we should use that.
> 
> Note that we do the same for many other sections of S&S that describe multiple spaces. If an item in the section refers to a particular space, we always refer in pi-base to that particular space number when referring to that item.

Done. Merging is blocked until you approve the implementation of your change.

**Review by prabau — APPROVED** [review]

**Pull request merged**
